readme/drug.md

```diff
@@ -2,36 +2,14 @@
 
 The `drug` subfolder allows you to import Odoo products provided as CSV files.
 It is designed specifically load OpenMRS Initializer drug type of files, ie, of a format compatible with [OpenMRS Module Initializer](https://github.com/mekomsolutions/openmrs-module-initializer/) concepts files.
-See an example file: [OpenMRS Drugs.csv](https://github.com/mekomsolutions/openmrs-module-initializer/blob/master/readme/drugs.md)
-
-The drug loader has been designed to allow OpenMRS and Odoo to share a common set of metadata and avoid maintaining 2 files.
-
-So the Odoo drug loader allows you to provide to import products (drugs) in Odoo and provide a base price directly on the OpenMRS file.
 
 The drug loader will apply a specific mapping to translate the OpenMRS Initializer file headers and into Odoo compatible headers such as:
 
 [drug_loader.py#L6-L11](https://github.com/mekomsolutions/odoo-initializer/blob/ce149ea01c58c1101ee43f20d5dbcbad26a332af/odoo_initializer/models/drug_loader.py#L6-L11)
 ```
 field_mapping = {
-    "product_variant_ids/uuid": "Uuid",
-    "name": "Name",
-    "lst_price": "odoo_price",
-    "product_variant_ids/categ_id/id": "odoo_category",
-    "type":"odoo_type",
-    "id": "odoo_id",
-    "uom_id/id": "odoo_uom",
-    "standard_price": "odoo_cost"
+    "product_variant_ids/id": "odoo_id",
+    "id": "odoo_id"
```

Comment:
Should we make the change and support 2 non-identical fields?
Maybe `odoo_variant_id`

Reply:
won't be used anymore as we will only load the drugs via the variants model.